app/models/manageiq/providers/ansible_tower/inventory_collection_default/automation_manager.rb

```diff
@@ -1,3 +0,0 @@
-class ManageIQ::Providers::AnsibleTower::InventoryCollectionDefault::AutomationManager < ManagerRefresh::InventoryCollectionDefault
```

Comment:
nope, this is all done in `app/models/manager_refresh/inventory/automation_manager.rb` and `app/models/manageiq/providers/ansible_tower/shared/inventory/persister/automation_manager.rb`